TAF_EasyRest/TestFramework/TestFramework.csproj

```diff
@@ -20,8 +20,4 @@
     <PackageReference Include="Selenium.WebDriver.ChromeDriver" Version="103.0.5060.5300" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
I'm not sure about that change. In fact, you have that folder, but you are removing it from the solution. I think you should revert this.